components/nvs_flash/include/nvs.h

```diff
@@ -54,6 +54,7 @@ typedef uint32_t nvs_handle;
 #define ESP_ERR_NVS_KEYS_NOT_INITIALIZED    (ESP_ERR_NVS_BASE + 0x16)  /*!< NVS key partition is uninitialized */
 #define ESP_ERR_NVS_CORRUPT_KEY_PART        (ESP_ERR_NVS_BASE + 0x17)  /*!< NVS key partition is corrupt */
 
+#define ESP_ERR_NVS_COMPARE_IDENTICAL       (ESP_ERR_NVS_BASE + 0x18)  /*!< NVS key is identical in comparison */
```

Comment:
Can you please add same comment as for `ESP_ERR_NVS_PAGE_FULL`, that this is never returned by the API?